python/ray/air/tests/test_api.py

```diff
@@ -143,7 +143,6 @@ def test_scaling_config_validate_config_bad_allowed_keys():
 @pytest.mark.parametrize(
     "trainer_resources", [None, {}, {"CPU": 1}, {"CPU": 2, "GPU": 1}, {"CPU": 0}]
 )
-@pytest.mark.parametrize("num_workers", [None, 1, 2])
```

Comment:
- `num_workers=None` is not allowed any more.
- `num_workers=1` is not possible for us to reconstruct the scaling config, since we cannot separate trainer resource from the merged bundle.